conftest.py

```diff
@@ -389,6 +389,13 @@ def t3k_device_mesh(request, silicon_arch_name, silicon_arch_wormhole_b0):
     logger.debug(f"multidevice with {device_mesh.get_num_devices()} devices is created")
     yield device_mesh
 
+    import tt_lib as ttl
+
+    for i in device_mesh.get_device_ids():
+        device = device_mesh.get_device(i)
+        ttl.device.DumpDeviceProfiler(device, True)
```

Comment:
This is going to run for every test, hope this is ok
cc: @tapspatel @mo-tenstorrent

Reply:
Yeah, in normal builds this is a no-op